build/Shared/HashCodeCombiner.cs

```diff
@@ -97,10 +91,11 @@ internal void AddSequence<T>(IEnumerable<T> sequence)
         internal static int GetHashCode(params object[] objects)
```

Comment:
K, just making sure wasn't missing a calling conflict due to precedence.

Can fix